src/common/components/select-repository-list/index.js

```diff
@@ -196,9 +184,9 @@ export class SelectRepositoryListComponent extends Component {
               { selectedRepositories.length } selected
             </strong>
             {' '}
-            (<a href="#" onClick={this.onHelpClick.bind(this)}>
+            (<Button appearance={ 'link' } onClick={this.onHelpClick.bind(this)} aria-label={ this.state.showMissingReposInfo ? 'Click to close the missing / private repos dialog' : 'Click to open the missing / private repos dialog' }>
```

Comment:
@richmccartney To expand on what I said yesterday, I think this really is a link, an in-page link that navigates to/from part of the page containing the explanation of why the list might be incomplete. It just so happens that in the current design, the explanation temporarily covers the list, instead of being (for example) at the bottom of the page.

But whether it is a link or a button, @bartaz appears to be correct: `aria-label` is not appropriate here. Besides [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute), [the W3C](https://www.w3.org/TR/WCAG20-TECHS/ARIA14.html), [Microsoft](https://msdn.microsoft.com/en-us/library/hh968243(v=vs.85).aspx), and [Opera](https://dev.opera.com/articles/ux-accessibility-aria-label/) are all clear that `aria-label` is for use on elements that don’t already have understandable text labels (for example, a close button where the label is only “X”). Furthermore, [Google’s reference](https://developers.google.com/web/fundamentals/accessibility/semantics-aria/aria-labels-and-relationships) illustrates that using it to give a button supplementary information such as “the size of the document and the type of file” would be incorrect, because “if a button has both text content and an aria-label, only the aria-label value will be used”.

I think that in context, “Any missing?” and “Return to repos list” are understandable text labels. If they aren’t, let’s fix the actual labels, rather than adding substitute labels that take effect only if you’re using an assistive device.